.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,14 +1,14 @@
 blank_issues_enabled: false
 contact_links:
-  - name: 💬 Discussion
-    url: https://github.com/ScratchAddons/ScratchAddons/discussions/new?category_id=16783727
-    about: Needs discussion (moved to Discussions)
-  - name: ❓ Help/questions
-    url: https://github.com/ScratchAddons/ScratchAddons/discussions/new?category_id=16783729
-    about: Ask a question (moved to Discussions)
+  - name: 💬 GitHub Discussions
+    url: https://github.com/ScratchAddons/ScratchAddons/discussions
+    about: Ask a question, or just discuss about Scratch Addons on our GitHub Discussions (our own forums)
   - name: 📝 Send feedback
     url: https://scratchaddons.com/feedback
     about: Send a general feedback on our website
-  - name: 💬 GitHub Discussions
-    url: https://github.com/ScratchAddons/ScratchAddons/discussions
-    about: Discuss about Scratch Addons on our GitHub Discussions (our own forums)
+  - name: 🏠 Community Discord
+    url: https://discord.gg/R5NBqwMjNc
+    about: Discuss about Scratch Addons on our community Discord server (recommended)
+  - name: 🔧 Development Discord
```

Comment:
Can we remove this and move everything into Community?

Reply:
How? To my knowledge we can't do anything on the new topic page. It's just go straight into [a page of boxes to fill](https://github.com/ScratchAddons/ScratchAddons/discussions/new).

Also, it wouldn't help people that wanted to create issues for requesting help. Also within the reason, please don't pin a topic and add the link there.

Reply:
No, I mean, removing links to Development Discord

Reply:
You sure really hate that moment, right?

I'm going to say no with that, dude. Most of us quite fine with them, I presume.